docker-hub/publish/insights-analytics.md

```diff
@@ -161,19 +159,19 @@ The following table describes the rules applied for determining intent behind
 pulls. To provide feedback or ask questions about these rules,
 [fill out the Google Form](https://forms.gle/nb7beTUQz9wzXy1b6){:
 target="_blank" rel="noopener" class="_"}.
-
+<!-- TODO: What do the first columns mean? -->
```

Comment:
iirc: the first column is the HTTP method, the second column is the image reference type (image tag name vs SHA 256 digest)